x-pack/plugins/security_solution/cypress/screens/alerts.ts

```diff
@@ -158,6 +157,13 @@ export const SHOW_TOP_N_HEADER =
 export const SHOW_TOP_N_CLOSE_BUTTON = '[data-test-subj="close"]';
 
 export const ALERTS_HISTOGRAM_LEGEND =
-  '[data-test-subj="alerts-histogram-panel"] [data-test-subj="withHoverActionsButton"]';
+  '[data-test-subj="alerts-histogram-panel"] .echLegendItem__action';
 
 export const SELECT_HISTOGRAM = '[data-test-subj="chart-select-trend"]';
+
+export const LEGEND_ACTIONS = {
+  ADD_TO_TIMELINE: '[data-test-subj="legend-New Rule Test-embeddable_addToTimeline"]',
```

Comment:
Any way you can pull in the `New Rule Test` from the rule itself in case it's removed or changed?